src/Features/CSharp/Portable/CodeFixes/Nullable/CSharpDeclareAsNullableCodeFixProvider.cs

```diff
@@ -77,10 +77,10 @@ private string GetEquivalenceKey(SyntaxNode node)
             return Task.CompletedTask;
         }
 
-        protected override bool IncludeDiagnosticDuringFixAll(FixAllState state, Diagnostic diagnostic, CancellationToken cancellationToken)
+        protected override bool IncludeDiagnosticDuringFixAll(Diagnostic diagnostic, Document document, string equivalenceKey, CancellationToken cancellationToken)
```

Comment:
Simple change to avoid passing down an internal FixAllState type into each code fix provider. This type is only meant for code fix engine, so we now only pass in the publically accessible entities to this method. This can potentially be extracted into a separate PR, but I don't see any value add given this is a very trivial change.

Reply:
Maybe call it `codeActionEquivalenceKey` to avoid looking like it's a diagnostic equivalence key?

Reply:
> This can potentially be extracted into a separate PR, but I don't see any value add given this is a very trivial change.

I'm fine with this if you confirm that this was just this mechanical change with no other logic changed.

Note: minor minor nit, but IDE preferred ordering would be: `Doucment, Diagnostic, EquivalenceKey`.  We like to go from 'wider to narrower' scope on the args for consistency.

Reply:
Yes, this was just a mechanical change. I will rename the parameter name to `codeActionEquivalenceKey`. I will retain the current order because almost all the overrides only need access to just the diagnostic and equivalence key. There is just one place where the override needs the syntax facts service. Hopefully once you move that service to shared analyzer layer, we can even remove the Document parameter completely. Let me track this cleanup in https://github.com/dotnet/roslyn/issues/41462